small/lsregion.c

```diff
@@ -30,6 +30,7 @@
  */
 
 #include "lsregion.h"
+#include <sys/uio.h>
```

Comment:
It is already in the `lsregion.h`, no need to duplicate it.

Reply:
That's an oversight. Thanks for noticing!